implants/lib/eldritch/src/file/list_impl.rs

```diff
@@ -208,12 +208,66 @@ pub fn list(starlark_heap: &Heap, path: String) -> Result<Vec<Dict>> {
 
 #[cfg(test)]
 mod tests {
+    use std::collections::HashMap;
+
+    use crate::runtime::Message;
+
     use super::*;
-    use tempfile::tempdir;
+    use pb::eldritch::Tome;
+    use tempfile::{tempdir, NamedTempFile};
+
+    fn init_logging() {
```

Comment:
nit: Perhaps it's time to create a logging crate

Reply:
I agree - can you add it as an issue for after ISTS.